vagrant/admin-build.sh

```diff
@@ -1,23 +1,34 @@
 #!/bin/bash
-
+set -xeuo pipefail
 # Install orchestrator
-rpm -i /tmp/orchestrator-release/orchestrator*.rpm
+if [[ -e /etc/redhat-release ]]; then
+  rpm -i /tmp/orchestrator-release/orchestrator*.rpm
+fi
+
+if [[ -e /etc/debian_version ]]; then
+  dpkg -i /tmp/orchestrator-release/orchestrator*.deb
+fi
+
+if [[ -e /orchestrator/vagrant/.sqlite ]]; then
+  cp -fv /usr/local/orchestrator/orchestrator-sample-sqlite.conf.json /etc/orchestrator.conf.json
+else
+  cp -fv /usr/local/orchestrator/orchestrator-sample.conf.json /etc/orchestrator.conf.json
+fi
 
 if [[ -e /etc/redhat-release ]]; then
 
   /sbin/chkconfig orchestrator on
-  cp /usr/local/orchestrator/orchestrator-sample.conf.json /etc/orchestrator.conf.json
   /sbin/service orchestrator start
 
 elif [[ -e /etc/debian_version ]]; then
 
   update-rc.d orchestrator defaults
-  cp /usr/local/orchestrator/orchestrator-sample.conf.json /etc/orchestrator.conf.json
   /usr/sbin/service orchestrator start
 
 fi
 
 echo '* * * * * root /usr/bin/orchestrator -c discover -i db1' > /etc/cron.d/orchestrator-discovery
 
 # Discover instances
-/usr/bin/orchestrator -c discover -i localhost
+/usr/bin/orchestrator --verbose --debug --stack -c redeploy-internal-db
```

Comment:
What happened that the above became necessary? Current logic should make it safe to run `orchestrator` without specifying `redeploy-internal-db`.

Reply:
ohh, sorry, i multiple run ```vagrant up``` and add redeploy-internal-db to try fix errors when using sqlite backend
i revert this changes right now